src/main/scala/newmodel/DocElement.scala

```diff
@@ -0,0 +1,75 @@
+package newmodel
```

Comment:
I created my own tree for now. I'm giving @xeno-by some time to finish the three top items on his roadmap https://github.com/scalameta/scalameta#roadmap. The last item is persistence of trees, this is the most helpful feature for me.

1) This week I will focus on rewriting scalakata and make stackoverflow executable. I will create a way to export and import between scalakata and stackoverflow. 
2) I will fork the most influential scala projects and provide them metadoc documentation (via sbt-plugin & travis-ci)
3) I will work on a ui prototype to assemble everything (projects, documentation, meta information, etc)
4) If I see a good traction, I will work on fp specific features like typesearch-a-la-hoogle. Signed builds published on bintray, etc. Basically I want to be Scala ecosystem janitor :-)